README.md

```diff
@@ -290,5 +290,49 @@ func DownloadFile(filepath string, url string) error {
 
 ```
 
-  
+### Possible to set Trim-box
+ 
+```go
+package main
+
+import (
+	"log"
+
+	"github.com/signintech/gopdf"
+)
+
+func main() {
+	
+    pdf := gopdf.GoPdf{}
+    mm6ToPx := 22.68
+    
+    // Base trim-box
+    pdf.Start(gopdf.Config{
+        PageSize: *gopdf.PageSizeA4, //595.28, 841.89 = A4
+        TrimBox: gopdf.Box{Left: mm6ToPx, Top: mm6ToPx, Right: mm6ToPx + 595, Bottom: mm6ToPx + 842},
```

Comment:
If I can understand correctly. I think it should be
```go
TrimBox:  gopdf.Box{Left: mm6ToPx, Top: mm6ToPx, Right: 595 - mm6ToPx, Bottom: 842 - mm6ToPx},
```

Reply:
Yes, you are right. I have just fixed.